webpack/components/extensions/Hosts/BulkActions/BulkPackagesWizard/BulkPackagesWizard.js

```diff
@@ -86,28 +87,28 @@ const BulkPackagesWizard = () => {
   const packageActionsNames = {
     install: __('Install packages'), remove: __('Remove packages'), upgrade: __('Upgrade packages'), upgradeAll: __('Upgrade packages'),
   };
-  const packageActions = () => {
-    switch (selectedAction) {
-    case INSTALL:
-      return (
-        <BulkPackagesInstallTable modalIsOpen={modalOpen} />
-      );
-    case REMOVE:
-      return (
-        <BulkPackagesRemoveTable modalIsOpen={modalOpen} />
-      );
-    default:
-      return (
-        <BulkPackagesUpgradeTable modalIsOpen={modalOpen} />
-      );
-    }
+
+
+  const initialSelectedHosts = fetchBulkParams();
+
+  const hostsBulkSelect =
+    useHostsBulkSelect({ initialSelectedHosts, modalIsOpen: modalOpen });
+
+  const currentlySelectedHosts =
+    hostsBulkSelect?.hostsResponse?.response?.results?.filter(host =>
+      !hostsBulkSelect.hostsBulkSelect.exclusionSet.has(host.id));
+
+  const getOSFamily = () => {
+    if (currentlySelectedHosts === undefined) { return BULK_ACTIONS_OSFAMILY.INVALID; } else if (currentlySelectedHosts[0]?.operatingsystem_family === 'Debian') { return BULK_ACTIONS_OSFAMILY.DEBIAN; }
```

Comment:
Conversely, when if statements get complex enough to have an `else if`, I strongly prefer them on multiple lines rather than single-line.

Reply:
I wonder why the linter does not automatically fix that one. Pretty sure I ran it several times over and it never complained about that - maybe even auto-corrected that one.